native-schema-locater/src/SchemaXmlFileLocater.ts

```diff
@@ -6,6 +6,8 @@
 import * as path from "path";
 import { DOMParser } from "@xmldom/xmldom";
 import { ECSchemaXmlContext } from "@itwin/core-backend";
+import { Logger, LogLevel } from "@itwin/core-bentley";
+import { NativeLoggerCategory } from "@bentley/imodeljs-native";

```

Comment:
you should be able to load this from core-backend because it reexports them.  I have seen issues with this in the past but at least try.  Having to reference the correct version of imodeljs-native is a real pain.

Reply:
Yep it worked. Thanks!